[product-json-to-csv]: Add a missing dependency

### Support reference number
SUPPORT-3190

### Description
When importing product mapping class from [product-json-to-csv](https://github.com/commercetools/nodejs/blob/master/packages/product-json-to-csv/src/map-product-data.js) I am getting this error:
```
Error: Cannot find module 'babel-runtime/core-js/object/entries'
```

Which says that there is a missing dependency in the module.

--- a/packages/product-json-to-csv/package.json
+++ b/packages/product-json-to-csv/package.json
@@ -42,6 +42,7 @@
     "@commercetools/sdk-middleware-user-agent": "1.2.10",
     "JSONStream": "1.3.2",
     "archiver": "2.1.1",
+    "babel-runtime": "^6.26.0",
     "bluebird": "3.5.1",
     "common-tags": "1.8.0",
     "flat": "4.1.0",

--- a/packages/product-json-to-csv/package.json
+++ b/packages/product-json-to-csv/package.json
@@ -42,7 +42,7 @@
     "@commercetools/sdk-middleware-user-agent": "1.2.10",
     "JSONStream": "1.3.2",
     "archiver": "2.1.1",
-    "babel-runtime": "^6.26.0",
+    "babel-runtime": "6.26.0",
     "bluebird": "3.5.1",
     "common-tags": "1.8.0",
     "flat": "4.1.0",

